functions/FunctionsExample/FunctionsExample/View/BackgroundFrame.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright 2022 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+struct BackgroundFrame<Content: View>: View {
+  var title: String
+  var description: String
+  let content: Content
+  let hitButton: () -> Void
+  init(title: String, description: String, hitButton: @escaping () -> Void,
+       @ViewBuilder content: () -> Content) {
+    self.title = title
+    self.description = description
+    self.content = content()
+    self.hitButton = hitButton
+  }
+
+  var body: some View {
+    VStack(alignment: .leading) {
+      Text(title)
+        .fontWeight(.bold)
+        .font(.title3)
+      Text(description)
+        .font(.subheadline)
+      ZStack {
+        RoundedRectangle(cornerRadius: 16)
+          .fill(Color(.secondarySystemBackground))
+          .frame(height: 150)
+        content
+      }
+      CustomStyledButton(title: "Run", action: hitButton)
+    }
+    .padding()
+    .frame(width: ScreenDimensions.width * 0.95)
```

Comment:
What do things look like without this manual frame modifier?

Reply:
It looks like all frame could be removed. We can control the frame through padding.